tutorials/darcy_thermo_mech/doc/content/workshop/bib/tutorial.bib

```diff
@@ -18,11 +18,3 @@ @article{griffiths1997no
   url = {https://onlinelibrary.wiley.com/doi/abs/10.1002/%28SICI%291097-0363%2819970228%2924%3A4%3C393%3A%3AAID-FLD505%3E3.0.CO%3B2-O},
   year = {1997}
 }
-
-@article{jasak1996error,
```

Comment:
it s used

Reply:
Indeed. For some reason it was building without throwing an error on my local machine as well as rendering correctly, but this was for the combined website build. More to come in this PR - just tossed this up early.